lib/screens/SearchResultsScreen.dart

```diff
@@ -6,15 +6,23 @@ import 'package:we_find/widgets/CourseWidget.dart';
 
 class SerchResultsScreen extends StatelessWidget {
   final List<CourseWrapped> _myCourses = [];
-  SerchResultsScreen(List<Course> courses) {
+  final String _searchQuery;
+  SerchResultsScreen(List<Course> courses, this._searchQuery) {
     for (Course eachCourse in courses) {
       // TODO: CHANGE THE LANGUAGE TO SOMETHING MEANINGFULL!
       _myCourses.add(CourseWrapped(eachCourse, Lang.DE));
     }
   }
 
-  ListView _buildSearchResults() {
+  ListView _buildSearchResults(ThemeData themeData) {
```

Comment:
Please pass the `BuildContext` as a parameter here, it is going to be needed when accessing the providers I set up

Reply:
I think it will be best to resolve this in a separate issue as this is not the only place I do this. I can stir up a fix right after merging.